submissions/onyemeMichael/to-do-app/src/components/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect} from 'react';
+import TodoForm from './TodoForm';
+import Todo from './Todo';
+
+function TodoList() {
+  const [todosLeft,setTodosRemaining] = useState(0);
+  const [todos, setTodos] = useState([]);
+  useEffect(() => {
+    setTodosRemaining(todos.filter(todo => !todo.isCompleted).length);
+    
+  },
+[ todos])
+   
+
+  const addTodo = todo => {
+    if (!todo.text || /^\s*$/.test(todo.text)) {
+      return;
+    }
+
+    const newTodos = [todo, ...todos];
+
+    setTodos(newTodos);
+  };
+
+  const updateTodo = (todoId, newValue) => {
+    if (!newValue.text || /^\s*$/.test(newValue.text)) {
+      return;
+    }
+
+    setTodos(prev => prev.map(item => (item.id === todoId ? newValue : item)));
```

Comment:
Instead of using `prev` why not access `todos` directly?

Reply:
Thanks